WordPress/Classes/ViewRelated/Plugins/PluginListCell.swift

```diff
@@ -23,6 +23,14 @@ class PluginListCell: UITableViewCell {
             view.leadingAnchor.constraint(greaterThanOrEqualTo: accessoryViewContainer.leadingAnchor).isActive = true
         }
     }
+
+    override func layoutSubviews() {
```

Comment:
more of a question rather than a nitpick, but: why the override of `layoutSubview` here, instead of `awakeFromNib` or something else?

Reply:
tbh I copy & pasted from elsewhere in the app then edited. Since it's _only_ colors, I think you're right that `awakeFromNib` is a better fit 👍